.infrastructure/helm-chart/todoapp/charts/mysql/templates/statefulSet.yml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{.Chart.Name}}
+  namespace: {{.Values.namespace}}
+spec:
+  selector:
+    matchLabels:
+      app: {{.Chart.Name}}
+  replicas: {{.Values.mysql.statefulSet.replicas}}
```

Comment:
Ensure that the `mysql.statefulSet.replicas` value is defined in your `values.yaml` file. If it's not defined, this could lead to issues when deploying the chart. Consider providing a default value or handling the case where it might be undefined.